test/IISExpress.FunctionalTests/InProcess/WebSocketTests.cs

```diff
@@ -20,7 +20,7 @@ public class WebSocketsTests
 
         public WebSocketsTests(IISTestSiteFixture fixture)
         {
-            _webSocketUri = fixture.BaseUri.Replace("http:", "ws:");
+            _webSocketUri = fixture.DeploymentResult.ApplicationBaseUri.Replace("http:", "ws:");
```

Comment:
Seems like a random fix.

Reply:
I removed extra properties from fixture to make it lazy.